src/embed/sync.rs

```diff
@@ -10,7 +10,22 @@ pub async fn begin_sync(
     fetch_client: &fetch::client::Client,
     begin_sync_req: &BeginSyncRequest,
```

Comment:
Is there a reason to pass this by reference? It seems like you can avoid cloning data_layer_auth below if you take ownership of this param.

Reply:
done in https://github.com/rocicorp/repc/pull/193